### PR TITLE
fix(signup): fix glitch

### PR DESCRIPTION
**Pull request by molimauro**
**What this PR does** 📖
When registering there is a small amount of time in which the login page (not the signup) is shown, causing a bad UX. This should fix that.

**Which issue(s) this PR fixes** 🔨

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

**Comment by josephmcg** [comment]
I can't login on this branch
image

File: pages/auth/unlock/index.vue [hunk 1]
```diff
@@ -14,13 +14,17 @@ export default Vue.extend({
...
+  } {
     return {
       pin: '',
+      status: 'idle',
       error: '',
-      decrypting: false,
-      peer: null,
-      showChangeLog: false,
+      step: 'signup',
     }
```
Comment (josephmcg) [inline pages/auth/unlock/index.vue:28]: I typically just cast the data as type
`status: 'idle' as 'idle' | 'loading'`

Comment (molimauro) [reply pages/auth/unlock/index.vue:28]: Do you think its too verbose? I'll change then for consistency :)

Comment (josephmcg) [reply pages/auth/unlock/index.vue:28]: I think either way is ok, I prefer fewer lines whenever possible. less scrolling lol

**Pull request merged**